### ISSUE TITLE
Fix text color contrast globally

### ISSUE THREAD

**Issue by vgarleanu** [Dusk-Labs/dim#132]
Dim isn't doing too great on colors when it comes to visual accessibility, right now thinking of instead of using white on orange, to use black on orange instead. Still looks good and much better contrast ratio.
